src/core/hle/service/hid/controllers/npad.cpp

```diff
@@ -500,17 +500,24 @@ void Controller_NPad::SetNpadMode(u32 npad_id, NPadAssignments assignment_mode)
     }
 }
 
-void Controller_NPad::VibrateController(const std::vector<u32>& controller_ids,
+void Controller_NPad::VibrateController(const std::vector<ControllerIds>& controller_ids,
                                         const std::vector<Vibration>& vibrations) {
-    LOG_DEBUG(Service_HID, "(STUBBED) called");
-
+    LOG_TRACE(Service_HID, "called");
```

Comment:
Leave the newline intact
```suggestion
    LOG_TRACE(Service_HID, "called");
    
```